src/features/Entities/Sensor/SensorManager.js

```diff
@@ -246,7 +246,9 @@ export class SensorManager {
 
   addNewSensor = async (macAddress, logInterval, logDelay, batteryLevel) => {
     const id = this.utils.uuid();
+    const name = id.slice(0, 8);
```

Comment:
mSupply schema was complaining about logs without names. Updated to assign an arbitrary name on creation based on mac address (user can always change this if they wish).